op-e2e/README.md

```diff
@@ -19,3 +19,8 @@ runtime, but others cannot be changed or else it will result in
 broken tests. Any changes to `devnetL1.json` should result in
 rebuilding the `.devnet` artifacts before the new values will
 be present in the `op-e2e` tests.
+
+### Troubleshooting
+If you encounter errors:
+* ensure you have the latest version of foundry installed (run `pnpm update:foundry` or `foundryup`)
```

Comment:
Currently `pnpm update:foundry` runs a [custom install script which installs the committed version ](https://github.com/ethereum-optimism/optimism/blob/0000000000000000000000000000000000000000:/ops/scripts/install-foundry.sh#L8-L9)of foundry, whereas `foundryup` will get install the latest nightly. 

I'm not sure which one is best to put here. For me, `foundryup` didn't require building anything from source, and fixed the problem 🤷 ?

Reply:
Not sure either. Maybe @clabby can suggest what to use here?

Reply:
We should use `pnpm update:foundry` to install the foundry commit specified in `versions.json`. Foundry uses nightly releases which are identified by their commit hash, and these stick around for 3 days before being pruned, with the exception of first nightly of every month which is persisted. The install script will look for a released binary of the `versions.json` commit and use it if present, but if there is no available binary it will build from source

Reply:
[532a9cd](https://github.com/ethereum-optimism/optimism/pull/9380/commits/532a9cdda2610ec679cd55fd70159ea829983af5)